<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NautilusCommander</class>
 <widget class="QWidget" name="NautilusCommander">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Nautilus Commander</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>658</y>
     <width>555</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(140, 140, 140);border: 0px; border-radius:6px;
font: bold 13pt;</string>
   </property>
   <property name="text">
    <string>OPEN MISSION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>465</y>
     <width>555</width>
     <height>85</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;border: 20px; border-radius:6px;
border-color: rgb(0, 0, 0);
font: bold 12pt;
</string>
   </property>
   <property name="text">
    <string>START NEW MISSION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOff">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>758</y>
     <width>555</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);border: 0px; border-radius:6px;
font: bold 13pt;</string>
   </property>
   <property name="text">
    <string>TURN OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/nautilus128x128.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>270</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 13pt</string>
   </property>
   <property name="text">
    <string>Welcome to Nautilus Commander</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>400</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 153, 153);
font: bold 11pt;</string>
   </property>
   <property name="text">
    <string>Â¿What do you want to do?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>600</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 153, 153);
font: bold 11pt;</string>
   </property>
   <property name="text">
    <string>Or</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>600</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: bold 15pt</string>
   </property>
   <property name="text">
    <string>NAUTILUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>620</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt</string>
   </property>
   <property name="text">
    <string>COMMANDER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/nautilusBack1920x1080.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>910</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 153, 153);
font: bold 10pt;</string>
   </property>
   <property name="text">
    <string>Need</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHelp">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>910</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 153, 153);
font: bold 10pt;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>Help?</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>340</y>
     <width>591</width>
     <height>601</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btnOpen</zorder>
  <zorder>btnNew</zorder>
  <zorder>btnOff</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>lblHelp</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>btnNew</tabstop>
  <tabstop>btnOpen</tabstop>
  <tabstop>btnOff</tabstop>
  <tabstop>lblHelp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
